### PR TITLE
EWPP-134: Drop usage of legacy site and page header.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by nagyad — DISMISSED** [review]

File: tests/Kernel/fixtures/rendering.yml [hunk 1]
```diff
@@ -1241,26 +1241,24 @@
...
-      '.ecl-page-header': 1
-      '.ecl-breadcrumb__segment': 2
+      '.ecl-page-header-core': 1
+      '.ecl-breadcrumb-core__segment': 2
       '.ecl-lang-select-page button.ecl-button': 1
       '.ecl-lang-select-page .ecl-expandable__content ul li a.ecl-link': 2
-      '.ecl-page-header__meta-list': 1
-      '.ecl-page-header__info-list': 1
-      '.ecl-page-header__info-item': 2
+      '.ecl-page-header-core__meta': 1
```
Comment (22Alexandra) [inline tests/Kernel/fixtures/rendering.yml:1342]: Can we add more items? We should test the rendering for all the possible classes for breadcrumb items:
`<li class="ecl-breadcrumb-core__segment" data-ecl-breadcrumb-core-item="static">`
`<li class="ecl-breadcrumb-core__segment ecl-breadcrumb-core__segment--ellipsis" data-ecl-breadcrumb-core-ellipsis>`
`<li class="ecl-breadcrumb-core__segment" data-ecl-breadcrumb-core-item="expandable">`
`<li class="ecl-breadcrumb-core__segment ecl-breadcrumb-core__current-page" data-ecl-breadcrumb-core-item="static" aria-current="page">`

File: tests/Behat/MinkContext.php [hunk 2]
```diff
@@ -174,7 +174,7 @@ protected function getLanguageSwitcherOverlay(): NodeElement {
    * @throws \Behat\Mink\Exception\ElementNotFoundException
    */
   protected function getBreadcrumb(): NodeElement {
-    $selector = 'nav.ecl-page-header__breadcrumb';
+    $selector = 'nav.ecl-page-header-core__breadcrumb';
```
Comment (22Alexandra) [inline tests/Behat/MinkContext.php:177]: We're missing this `data-ecl-auto-init="BreadcrumbCore"` from the nav tag: https://ecl-twig-js.netlify.app/ec/?path=/story/components-page-headers-page-header-core--title

File: templates/patterns/page_header/page_header.ui_patterns.yml [hunk 3]
```diff
@@ -49,8 +49,8 @@ page_header:
           label: "Announcements"
     infos:
       type: "array"
-      label: "Infos"
-      description: "A list of infos of the current page."
+      label: "Infos (deprecated)"
+      description: "A list of infos of the current page. This was deprecated from ECL 2.30."
```
Comment (22Alexandra) [inline templates/patterns/page_header/page_header.ui_patterns.yml:61]: deprecated
In other deprecated patterns we only mentioned the major version of ecl. We could change to This was deprecated in ECL 2.x.

**Comment by 22Alexandra** [comment]
Seems like the distance between language selector and search box is higher than here https://ecl-twig-js.netlify.app/ec/?path=/story/components-site-headers-core--default
language selector

**Comment by 22Alexandra** [comment]
There is a test failing related to the language selector.

**Comment by 22Alexandra** [comment]
> Seems like the distance between language selector and search box is higher than here https://ecl-twig-js.netlify.app/ec/?path=/story/components-site-headers-core--default
> language selector

Fixed.

**Comment by 22Alexandra** [comment]
> There is a test failing related to the language selector.

Fixed.

**Review by 22Alexandra — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

File: templates/patterns/page_header/page_header.ui_patterns.yml (hunk 3 above)
Comment (sergepavle) [reply templates/patterns/page_header/page_header.ui_patterns.yml:61]: Mentioning that this element deprecated from ECL 2.x meant really that such element deprecated from version 2.x (during upgrade from ECL 1 to ECL 2). In this particular case, ECL really doesn't support this element anymore only from 2.30 version.

File: tests/Behat/MinkContext.php (hunk 2 above)
Comment (sergepavle) [reply tests/Behat/MinkContext.php:177]: We have such an attribute when we have more than 3 breadcrumb elements. This logic implemented in the code of the ecl-twig component.

**Review by ademarco — DISMISSED** [review]

**Review by ademarco — DISMISSED** [review]

**Review by ademarco — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**